.travis.yml

```diff
@@ -29,19 +29,18 @@ os:
   - osx
   - windows
 julia:
-  - 1.0  # LTS
-  - 1.4  # Latest release
+  - 1.4  # Latest stable release
+  - 1.5  # Latest release
   - nightly
 arch:
   - x64
   - x86
+
 jobs:
+  allow_failures:
+    - julia: nightly
```

Comment:
I disallow nightly failures on this package as I want to know immediately if there is an issue with this package and the latest version of Julia

Reply:
So I should comment it out? 
Ok, I didn't want to make PR with CI crashing, but if you're ok with CI showing "failed" just because it fails on nightly, I don't have a problem with that.

Reply:
I'd just leave this how it was. I'll fix the nightly issue in another PR

Reply:
Ok, I commented it out, it passes also on nightly now, nice.